### PR TITLE
#34 Feature:  A seller should be able to delete item

### PR DESCRIPTION
**Pull request by yobuir**
#### What does this PR do?
A seller should be able to delete a specific item from their collection when it's no longer needed

#### Description of Task to be completed?
Seller should be able to delete a specific item from collection when it's
no longer on offer 
- When the seller clicks on delete item a confirm dialogue box should popup to confirm item deletion

#### How should this be manually tested?
```
git clone https://github.com/atlp-rwanda/ecommerce-app-predators-fe.git
cd ecommerce-app-predators-fe
git checkout branch-?
npm i
npm run dev
```

#### What are the relevant pivotal tracker/Trello stories?
Trello ([link](https://trello.com/c/I1ntJ3hD/34-a-seller-should-be-able-to-delete-a-specific-item-from-their-collection))

#### Screenshots (if appropriate)
Screenshot (78)

### CODE REVIEW

**Review by Muhet — CHANGES_REQUESTED** [review]
Can you fix those conflict and also you test are not passing

**Review by faid-terence — APPROVED** [review]

**Review by ALVINdimpos — APPROVED** [review]

**Review by vanessa-abewe — APPROVED** [review]

File: src/components/notification/Notification.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import{ useState } from 'react'
+
+function Notification(props: any) {
```
Comment (chanceDira) [inline src/components/notification/Notification.tsx]: Create an interface here, Let's not use "any"

File: src/components/sample.tsx [hunk 2]
```diff
@@ -1,3 +1,3 @@
 export const add = (a: any, b: any) => {
```
Comment (chanceDira) [inline src/components/sample.tsx]: Create interface

File: src/components/sidebar/Sidebar.tsx [hunk 3]
```diff
@@ -1,18 +1,13 @@
...
 import electron from '../../assets/sidebar/Electron.svg';
-import { useState } from 'react';
 
-export default function Sidebar() {
-    const [isOpen, menu] = useState(false);
 
-    const handleCloseMenu = () => {
-        menu(!isOpen)
-    }
+export default function Sidebar(props: any) {
```
Comment (chanceDira) [inline src/components/sidebar/Sidebar.tsx]: create interface

File: src/redux/action/ProductAction.tsx [hunk 4]
```diff
@@ -1,18 +1,43 @@
...
       return response.data;
     } catch (error) {
       throw new Error('error'); //Something went wrong!
     }
   }
 );
  
+export const productRemove = createAsyncThunk(
+  'products/productRemove',
+  async ({data}: any, thunkAPI) => {
```
Comment (chanceDira) [inline src/redux/action/ProductAction.tsx]: remove "any"

**Review by chanceDira — CHANGES_REQUESTED** [review]
Nice work @yobuir 
Below are some suggested changes

**Review by chanceDira — APPROVED** [review]
Looking good

**Review by Muhet — APPROVED** [review]
great progress

**Pull request merged**